modules/weko-records-ui/weko_records_ui/pdf.py

```diff
@@ -221,7 +221,9 @@ def get_oa_policy(activity_id):
         or header_display_position is None
     ):
         positions['str_position'] = 'C'
-        x, y = get_center_position(header_output_image_name)
+        x = 0
+        if header_output_image_name:
```

Comment:
The var y is not used or access so we can changed this to
x = get_center_position(header_output_image_name) if header_output_image_name else 0